aeron-client/src/main/c/aeron_client_conductor.c

```diff
@@ -2085,15 +2085,16 @@ int aeron_client_conductor_offer_remove_command(
     return 0;
 }
 
-int aeron_client_conductor_offer_destination_command(
+int64_t aeron_client_conductor_offer_destination_command(
```

Comment:
Style wise should we always return `int` for success/failure code or dual purpose with the correlation id? We could take an arg that is a `uint64_t *` for the correlation id.  Be good to discuss.

Reply:
I would say `int` return and take a `int64_t *` to return the id. This is an internal matter. But could be easier.

Reply:
I've gone with the return int for success/failure and using a pointer for the correlation_id.  I've also surfaced that to the API methods that call down from there e.g. `aeron_exclusive_publication_add_destination` so the style is consistent.